doc/WebSite/Validating-Data-Transfer-Objects.md

```diff
@@ -124,10 +124,10 @@ use these attributes to control validation:
 ### Normalization
 
 We may need to perform an extra operation to arrange DTO parameters
```

Comment:
How about: "perform extra operation`s` to `prepare the` DTO parameters"

It's difficult to find a good synonym of "normalize" 😕